pkg/kubectl/cmd/util/helpers.go

```diff
@@ -455,3 +456,39 @@ func UpdateObject(info *resource.Info, updateFn func(runtime.Object) error) (run
 
 	return info.Object, nil
 }
+
+func GetThirdPartyGroupVersions(c *client.RESTClient) ([]unversioned.GroupVersion, []unversioned.GroupVersionKind, error) {
+	result := []unversioned.GroupVersion{}
+	gvks := []unversioned.GroupVersionKind{}
+	thirdPartyList := unversioned.APIGroupList{}
+	err := c.Get().AbsPath("/apis").Do().Into(&thirdPartyList)
+	if err != nil {
+		if !errors.IsForbidden(err) && !errors.IsNotFound(err) {
+			return nil, nil, err
+		}
+	} else {
+		for ix := range thirdPartyList.Groups {
+			group := &thirdPartyList.Groups[ix]
+			if group.Name == "extensions" {
```

Comment:
This doesn't look right.  Special casing particular groups isn't a good idea.  Given my suspicion of your intent (all GroupVersions that aren't compiled in?), I could see checking whether the group is registered/enabled.